<a href="https://colab.research.google.com/github/pedroallmeiida/econometria/blob/main/Exemplos%20no%20R/Exemplo3_modelo_lin_log.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

# Modelo Lin-Log

## Exemplo 3

Suponha agora que queremos conhecer a varição absoluta de Y da uma variação percentual de X. Neste exemplo vamos estudar o comportamento das despesas com alimentação na índia usando sua moeda local (rubia).

**Significado das Variáveis:**

- Despesas_Alimentacao: Despesas com alimentação (em rubias)
- Despesa_Total: Despesas totais (em rubias)

In [2]:
# Bibliotecas R
library(ggplot2)
library(readr)


Leitura do dataset

In [3]:
df = read.csv( "https://raw.githubusercontent.com/pedroallmeiida/econometria/refs/heads/main/Dados/despesas_alimentacao.csv" )
df

Despesas_Alimentacao Despesa_Total
1  217                  382          
2  196                  388          
3  303                  391          
4  270                  415          
5  325                  456          
6  260                  460          
7  300                  472          
8  325                  478          
9  336                  494          
10 345                  516          
11 325                  525          
12 362                  554          
13 315                  575          
14 355                  579          
15 325                  585          
16 370                  586          
17 390                  590          
18 420                  608          
19 410                  610          
20 383                  616          
21 315                  618          
22 267                  623          
23 420                  627          
24 300                  630          
25 410                  635          
26 220                  640          
27 403                  648          
28 350                  650          
29 390                  655          
30 385                  662          
31 470                  663          
32 322                  677          
33 540                  680          
34 433                  690          
35 295                  695          
36 340                  695          
37 500                  695          
38 450                  720          
39 415                  721          
40 540                  730          
41 360                  731          
42 450                  733          
43 395                  745          
44 430                  751          
45 332                  752          
46 397                  752          
47 446                  769          
48 480                  773          
49 352                  773          
50 410                  775          
51 380                  785          
52 610                  788          
53 530                  790          
54 360                  795          
55 305                  801

Modelo Lin-Log

In [4]:
lm_linlog<- lm( Despesas_Alimentacao~  log(Despesa_Total), data = df)
summary(lm_linlog)


Call:
lm(formula = Despesas_Alimentacao ~ log(Despesa_Total), data = df)

Residuals:
    Min      1Q  Median      3Q     Max 
-158.43  -43.42    5.61   34.50  178.05 

Coefficients:
                   Estimate Std. Error t value Pr(>|t|)    
(Intercept)        -1283.91     292.81  -4.385 5.54e-05 ***
log(Despesa_Total)   257.27      45.43   5.662 6.20e-07 ***
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Residual standard error: 66.48 on 53 degrees of freedom
Multiple R-squared:  0.3769,	Adjusted R-squared:  0.3652 
F-statistic: 32.06 on 1 and 53 DF,  p-value: 6.201e-07


**Intrepretação do modelo log-log:**

A Equação do modelo predito é dada por:

$$
\widehat{\text{Despesas_Alimentacao}}_i = -1283.91 + 257.27 \, \ln \text{Despesa_Total}_i
$$

em que as estimativas para o intercepto e coeficiente de regressão são dados respectivamente por, $\hat\beta_0 = -1283.91$ e $\hat\beta_1 = 257.27$.

Seja $\hat\beta_1 = 257.27$, significa que um aumento de 1%, em média, nas despesas totais leva a um aumento de cerca de 2.57 rupias nas despesas com alimentos das 55 famílias incluídas na amostra.